tests/ga/test_monitor.py

```diff
@@ -562,10 +644,16 @@ def test_collect_and_send_events(self, mock_lib_dir, patch_send_event, *args):
                          '<Param Name="SubscriptionId" Value="DummySubId" T="mt:wstr" />' \
                          '<Param Name="ResourceGroupName" Value="DummyRG" T="mt:wstr" />' \
                          '<Param Name="VMId" Value="DummyVmId" T="mt:wstr" />' \
-                         '<Param Name="ImageOrigin" Value="1" T="mt:uint64" />'\
-                         '<Param Name="ContainerId" Value="c6d5526c-5ac2-4200-b6e2-56f2b70c5ab2" T="mt:wstr" />]]>'\
+                         '<Param Name="ImageOrigin" Value="1" T="mt:uint64" />' \
+                         '<Param Name="ContainerId" Value="c6d5526c-5ac2-4200-b6e2-56f2b70c5ab2" T="mt:wstr" />' \
+                         '<Param Name="GAVersion" Value="WALinuxAgent-2.2.44" T="mt:wstr" />' \
+                         '<Param Name="EventTid" Value="" T="mt:wstr" />' \
+                         '<Param Name="EventPid" Value="" T="mt:wstr" />' \
+                         '<Param Name="TaskName" Value="" T="mt:wstr" />' \
+                         '<Param Name="KeywordName" Value="" T="mt:wstr" />]]>' \
                          '</Event>'.format(AGENT_VERSION)
 
+        self.maxDiff = None
```

Comment:
Unused?

Reply:
This is a flag used by the testing framework to limit how much it should present if there is a difference, on the console. I set the maxDiff=None to explicitly show the diff or else you would see a message like `Diff is 6619 characters long. Set self.maxDiff to None to see it.`